test/.eslintrc

```diff
@@ -0,0 +1,6 @@
+extends:
+  - '../.eslintrc'
+  - 'homeoffice/config/testing'
+
+env:
```

Comment:
I'm not sure dotfiles will do this but @JoeChapman would know better. I'll remove the env now